spacy/displacy/render.py

```diff
@@ -1,12 +1,15 @@
-from typing import Dict, Any, List, Optional, Union
 import uuid
+import itertools
+from typing import Any, Dict, List, Optional, Tuple, Union, Set
```

Comment:
Small nitpick: we typically put the `from typing` import statements on the very first line(s) of the file :-)

Reply:
Gotcha will do that!